apps/api-gateway/src/organization/organization.service.ts

```diff
@@ -51,12 +56,12 @@ export class OrganizationService extends BaseService {
    * @param
    * @returns Public organizations list
    */
-  async getPublicOrganizations(getAllOrgsDto: GetAllOrganizationsDto): Promise<{ response: object }> {
+  async getPublicOrganizations(getAllOrgsDto: GetAllOrganizationsDto): Promise<{ response: GetOrgs }> {
```

Comment:
Let's not have an object key "response" outside the interface. This should be a part of Response interface. Please update this at multiple places in this PR.